pom.xml

```diff
@@ -36,6 +36,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <java.version>1.8</java.version>
     <jmh.version>0.5.6</jmh.version>
+    <eclfw.version>7.0.0</eclfw.version>
```

Comment:
What does this stand for? Perhaps this could be also `eclipse.collections.version` for clarity.